### ISSUE TITLE
'attempt to add with overflow' while reading LAZ file with RGB

### ISSUE THREAD

**Issue by Mortano** [tmontaigu/laz-rs#13]
I encountered the following error while reading certain LAZ files that contain RGB colors:
`thread 'main' panicked at 'attempt to add with overflow', [omitted]/laz-0.1.0/src/las/rgb.rs:699:39`

This should be reproducible e.g. with files from the [PG&E Diablo Canyon Power Plant](http://opentopo.sdsc.edu/lidarDataset?opentopoID=OTLAS.032013.26910.2) dataset, such as this file: https://cloud.sdsc.edu/v1/AUTH_opentopography/PC_Bulk/CA13_SAN_SIM/ot_35120A4201B_1.laz 

I am using `las-rs` with the `laz` flag enabled:
```
[dependencies]
las = { version= "0.7.0", features=["laz"] }
```
I read the file like this:
```rust
let mut reader = Reader::from_path("./ot_35120A4201B_1.laz").unwrap();
let points = reader.points().map(|r| r.unwrap()).collect::<Vec<_>>();
```

Reading the file with `lasinfo` works fine.

**Comment by Mortano** [comment]
So upon further digging this seems to be fixed in version `0.3.0` of `laz-rs`, however `las-rs` is still using version `0.1.0`. I will post a request to update the `laz-rs` version in the `las-rs` project and close this issue.

**Comment by tmontaigu** [comment]
Yep, I fixed these overflow things. However they only appeared in debug builds because in release builds, cargo disable overflow checks (and the code code stills compresses and decompresses correctly)

so if you build in release mode `cargo build --release` then it will work without having to wait for an upgrade of las-rs
